src/AspNetCore/Src/applicationmanager.cxx

```diff
@@ -77,7 +124,9 @@ APPLICATION_MANAGER::GetApplication(
 Finished:
 
     if (fExclusiveLock == TRUE)
+    {
```

Comment:
Is there any reason why we can't do if (fExclusiveLock)?

Reply:
fixed